samples/shields/x_nucleo_iks01a3/sensorhub/README.rst

```diff
@@ -0,0 +1,80 @@
+.. _x-nucleo-iks01a3-shub-sample:
+
+X-NUCLEO-IKS01A3: shield (Mode 2) sample
+########################################
+
+Overview
+********
+This sample is provided as an example to test the X-NUCLEO-IKS01A3 shield
+configured in Sensor Hub mode (Mode 2).
+Please refer to :ref:`x-nucleo-iks01a3` for more info on this configuration.
+
+This sample enables LIS2DW12 and LSM6DSO sensors. Since all other shield
+devices are connected to LSM6DSO, the LSM6DSO driver is configured in sensorhub
+mode (CONFIG_LSM6DSO_SENSORHUB=y) with a selection of two maximum slaves
+among LPS22HH, HTS221 and LIS2MDL (default is LIS2MDL + LPS22HH).
+
+Then sensor data are displayed periodically
+
+- LIS2DW12 3-Axis acceleration
+- LSM6DSO 6-Axis acceleration and angular velocity
+- LSM6DSO (from LIS2MDL) 3-Axis magnetic field intensity
+- LSM6DSO (from LPS22HH) ambient temperature and atmospheric pressure
+
+Optionally HTS221 can substitute one between LIS2MDL and LPS22HH
+
+- LSM6DSO (from HTS221): ambient temperature and relative humidity
+
+
+Requirements
+************
+
+This sample communicates over I2C with the X-NUCLEO-IKS01A3 shield
+stacked on a board with an Arduino connector. The board's I2C must be
+configured for the I2C Arduino connector (both for pin muxing
+and device tree). See for example the :ref:`nucleo_f401re_board` board
+source code:
+
+- :file:`$ZEPHYR_BASE/boards/arm/nucleo_f401re/nucleo_f401re.dts`
+- :file:`$ZEPHYR_BASE/boards/arm/nucleo_f401re/pinmux.c`
+
+Please note that this sample can't be used with boards already supporting
+one of the sensors available on the shield (such as disco_l475_iot1) as zephyr
+does not yet support sensors multiple instances.
```

Comment:
This is a little deceiving. Zephyr can certainly support multiple sensor instances, it's the specific drivers that don't actually implement multiple instances.

Reply:
Actually this sentence was copied@paisted from other samples:
   https://docs.zephyrproject.org/latest/samples/shields/x_nucleo_iks01a1/README.html
https://docs.zephyrproject.org/latest/samples/shields/x_nucleo_iks01a2/README.html

But I can change it to "... as sensors multiple instances is not supported".

Reply:
I changed it in this way:

 ```
Please note that this sample can't be used with boards already supporting
 one of the sensors available on the shield (such as disco_l475_iot1)
 as sensors multiple instances are not supported.

```
@MaureenHelm 
Pls check my english and tell me if it is better than before. Thank you.